src/include/btree_inline.h

```diff
@@ -786,13 +786,13 @@ __wt_page_only_modify_set(WT_SESSION_IMPL *session, WT_PAGE *page)
      * going negative. Note that the checkpoint can only clean the page if it belongs to the
      * metadata or the history store.
      */
-    size = 0;
+    size = __wt_atomic_load_size_relaxed(&page->memory_footprint);
     if (WT_UNLIKELY(!WT_PAGE_IS_INTERNAL(page) &&
```

Comment:
A bit late to these PRs, but why the usage of WT_UNLIKELY? Is this path sufficiently a performance path?

Reply:
What is rare, taking this branch?